src/helix-ui/elements/HXDrawerElement.less

```diff
@@ -0,0 +1,183 @@
+@import "./HXElement";
+@import (reference) "components/buttons";
+
+#hxClose {
+  #Button.reset();
+  color: var(--hxClose-color, @gray-700);
+  font-size: 1rem;
+  height: 2.75rem; // 44px for a11y
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 0.375rem; // ~6px
+  width: 2.75rem; // 44px for a11y
+}
+
+#hxDrawer {
+  background-color: inherit;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  left: 100%;
+  overflow: hidden;
+  position: fixed;
+}
+
+/* ========== Width and Animation ========== */
+
+/*
+  Close Transitions
+  ---------------------------
+  transform   0.18s   ease-in
+  box-shadow  0.18s   linear
+*/
+#hxDrawer {
+  transition-duration: 0.18s;
+  transition-property: transform, box-shadow;
+  transition-timing-function: ease-in, linear;
```

Comment:
The `ease-in` value corresponds to the `transform` transition property above.

Similar with `linear` for the `box-shadow` transition property.